src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -257,10 +259,10 @@ public void SetVolume(float volume)
 
 	private void OnHtmlTimeUpdated(object sender, EventArgs e)
 	{
-		if (this.Log().IsEnabled(Uno.Foundation.Logging.LogLevel.Debug))
-		{
-			this.Log().Debug($"Time updated [{Source}]");
-		}
+		//if (this.Log().IsEnabled(Uno.Foundation.Logging.LogLevel.Debug))
+		//{
+		//	this.Log().Debug($"Time updated [{Source}]");
+		//}
```

Comment:
Probably best to keep? But have the loglevel.trace instead is probably better.

Reply:
I removed it because it was being triggered too many times.
Every change on the time of the player .